homeassistant/core.py

```diff
@@ -510,16 +511,35 @@ def create_task(self, target: Coroutine[Any, Any, Any]) -> None:
         self.loop.call_soon_threadsafe(self.async_create_task, target)
 
     @callback
-    def async_create_task(self, target: Coroutine[Any, Any, _R]) -> asyncio.Task[_R]:
```

Comment:
Should we allow this in `create_task` in case they need to set it up in another thread?

Reply:
let's see that when we get there. It feels weird that sync code would have an async background task ?